examples/3d/wireframe.rs

```diff
@@ -1,38 +1,45 @@
 //! Showcases wireframe rendering.
 
 use bevy::{
-    pbr::wireframe::{Wireframe, WireframeConfig, WireframePlugin},
+    pbr::wireframe::{Wireframe, WireframeColor, WireframeConfig, WireframePlugin},
     prelude::*,
     render::{render_resource::WgpuFeatures, settings::WgpuSettings},
 };
 
 fn main() {
     App::new()
+        // When rendering wireframes, you need to enable the POLYGON_MODE_LINE feature of wgpu
         .insert_resource(WgpuSettings {
```

Comment:
I assume that this needs to be inserted before DefaultPlugins is added? We should note that here if so

Reply:
This is fixed now with the current plugin group design